docs/curators/risk-reports/782143811495370029786561603242592541243660864649184692340475351040560976323/dog.mdx

```diff
@@ -0,0 +1,165 @@
+- [StarkGate Audit](https://github.com/NethermindEth/PublicAuditReports/blob/main/NM0064-DRAFT_STARKGATE.pdf)
+- [StarkGate Stats](https://dune.com/sapiens/starknet-insighttokenomics-and-bridge-stats)
+
+**Rating**
+
+The proposed rating is:
+
+- 🟩 Low
+
+
+## Oracle Risk
+
+The pool uses Pragma's `DOG/USD` price feed.
+The Pragma oracle is a native Starknet oracle system that secures over USD 200m across all major projects on Starknet.
+It is fully decentralized in that it does not rely on off-chain infrastructure. Instead, data providers directly push prices on-chain and aggregation is done by the data consumer directly.
+
+**Smart contract risk**
+
+Find an overview of the Pragma smart contract system [here](https://github.com/Astraly-Labs/pragma-oracle?tab=readme-ov-file#overview).
+
+The Pragma smart contracts have been audited by Nethermind and both the [source code](https://github.com/Astraly-Labs/pragma-oracle) and [audit report](https://github.com/astraly-labs/pragma-oracle/tree/main/audits) are publicly available.
+
+**Price composition & source diversity**
+
+The oracle price is computed as the Median over the most recent data points available for the various Pragma providers.
+
+**Oracle fail-safes**
+
+No autonomous oracle fail-safes and fallbacks are configured. CarmineDAO monitors and pauses pools manually in case of an emergency.
+
+**Datapoints**
+
+- [Pragma Oracle Contract](https://voyager.online/contract/0x02a85bd616f912537c50a49a4076db02c00b29b2cdc8a197ce92ed1837fa875b)
+- [Pragma Documentation](https://docs.pragma.build/Resources/Cairo%201/data-feeds/consuming-data)
+- [Pragma Pair IDs](https://docs.pragma.build/Resources/Cairo%201/data-feeds/supported-assets)
+- [Source Code](https://github.com/Astraly-Labs/pragma-oracle)
+- [Audit Report](https://github.com/astraly-labs/pragma-oracle/tree/main/audits)
+
+**Rating**
+
+The proposed rating is:
+
+- 🟩 Low
+
+
+## Market Risk
+
+`DOG` deposits can be borrowed with all pool assets as collateral.
+
+The pool enforces *debt caps* and *liquidation discounts* reflecting the on-chain liquidity available for liquidations.
+
+Liquidations, as per the used official factory extension (see Section *Pool Risk*), are public and prioritize rapid liquidations.
+
+The resulting market risk is computed using the [Standard Market Risk Model](https://docs.google.com/spreadsheets/d/1zU9IRKCX8xicSNd4UV78hMEw_rf-ZtGL/edit?usp=sharing&ouid=108086377410680948516&rtpof=true&sd=true).
+
+**Datapoints**
+
+- [Liquidation Documentation](https://docs.vesu.xyz)
+- [Standard Market Risk Model](https://docs.google.com/spreadsheets/d/1zU9IRKCX8xicSNd4UV78hMEw_rf-ZtGL/edit?usp=sharing&ouid=108086377410680948516&rtpof=true&sd=true)
+
+**Rating**
+
+The proposed rating is:
+
+- 🟨 Medium
+
+
+## Collateral Risk
+
+The pool enables borrowing of `DOG` with any of the pool's assets as collateral. Collateral *rehypothecation* is not enabled in the pool. 
```

Comment:
collateral rehypothecation in fact is enabled in the pool as the collateral used to secure DOG loans can again be borrowed by third parties